src/Microsoft.Framework.Logging/Logger.cs

```diff
@@ -38,16 +38,27 @@ public void Log(LogLevel logLevel, int eventId, object state, Exception exceptio
 
         public bool IsEnabled(LogLevel logLevel)
         {
-            return logLevel >= _loggerFactory.MinimumLevel && _loggers.Any(l => l.IsEnabled(logLevel));
+            if (logLevel < _loggerFactory.MinimumLevel)
+            {
+                return false;
+            }
+            foreach (var logger in _loggers)
+            {
+                if (!logger.IsEnabled(logLevel))
```

Comment:
Doesn't this change the original behavior? Or do we want to change the behavior.
Should be ```if(logger.IsEnabled(logLevel)) return true;``` return false at the end of the function because no logger was found for that logLevel.